### PR TITLE
add new term ocular dysgenesis caused by defects in PAX6 regulation

### LINKED ISSUE #7066: Request for new term [ocular dysgenesis caused by defects in PAX6 regulation]
### Label

ocular dysgenesis caused by defects in PAX6 regulation

### Synonyms

_No response_

### Synonym type

None

### Definition

Any eye disorder in which the cause of the disease is a variant in the _PAX6_ gene itself or a variant within another locus that results in defective regulation of the _PAX6_ gene (PMID: 24290376, PMID: 26394807, PMID: 17679951).

### Parent term

MONDO:0005328 eye disorder

### Children term(s)

MONDO:0014937 aniridia 2, MONDO:0800183 PAX6-related ocular dysgenesis, MONDO:0014938 aniridia 3

### ORCID Identifier

0000-0001-5935-2391

### Website URL

https://clinicalgenome.org/affiliation/40077/

### Additional comments

This new term is proposed by the ClinGen Glaucoma / Neuro-Ophthalmology GCEP as a parent term for cases of ocular disease caused by variants in the _PAX6_ gene itself (MONDO:0800183 PAX6-related ocular dysgenesis) or by disruption of PAX6 expression by variants in the _ELP4_ locus (MONDO:0014937 aniridia 2) or _TRIM44_ locus (MONDO:0014938 aniridia 3).

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
close #7066

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -329321,26 +329321,27 @@ property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/
...
 synonym: "aniridia 2" EXACT [OMIM:617141]
-synonym: "aniridia 2; AN2" EXACT []
 synonym: "aniridia type 2" EXACT [MONDORULE:1, OMIM:617141]
 xref: GARD:16200 {source="OMIM:617141"}
 xref: MESH:C536372 {source="MONDO:equivalentTo"}
 xref: OMIM:617141 {source="MONDO:equivalentTo"}
 xref: SCTID:253232000 {source="MONDO:equivalentTo"}
 xref: UMLS:C0344543 {source="MONDO:ncbi_mim2gene_medline", source="MONDO:notFoundInDiseaseSubset"}
 is_a: MONDO:0007119 {source="OMIM:617141", source="Orphanet:250923/btnt"} ! isolated aniridia
-relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/1171 {source="MONDO:mim2gene_medgen"} ! ELP4
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:329337]: I don't think it is correct to remove this axiom. ELP4 is reported in OMIM:617141 to be associated with this disease.

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -329354,10 +329355,9 @@ xref: GARD:16201 {source="OMIM:617142"}
 xref: OMIM:617142 {source="MONDO:equivalentTo"}
 xref: UMLS:C4310695 {source="MONDO:ncbi_mim2gene_medline", source="MONDO:notFoundInDiseaseSubset"}
 is_a: MONDO:0007119 {source="MONDO:Redundant", source="OMIM:617142", source="Orphanet:250923/btnt"} ! isolated aniridia
-intersection_of: MONDO:0007119 ! isolated aniridia
-intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/19016 ! TRIM44
-relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/19016 {source="MONDO:mim2gene_medgen"} ! TRIM44
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:329359]: It isn't correct to remove this axiom. TRIM44 is reported in OMIM:617142 to be associated with this disease.

File: src/ontology/mondo-edit.obo [hunk 3]
```diff
@@ -329354,10 +329355,9 @@ xref: GARD:16201 {source="OMIM:617142"}
 xref: OMIM:617142 {source="MONDO:equivalentTo"}
 xref: UMLS:C4310695 {source="MONDO:ncbi_mim2gene_medline", source="MONDO:notFoundInDiseaseSubset"}
 is_a: MONDO:0007119 {source="MONDO:Redundant", source="OMIM:617142", source="Orphanet:250923/btnt"} ! isolated aniridia
-intersection_of: MONDO:0007119 ! isolated aniridia
-intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/19016 ! TRIM44
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:329358]: It isn't correct to remove this axiom. TRIM44 is reported in OMIM:617142 to be associated with this disease.

**Review by sabrinatoro — CHANGES_REQUESTED** [review]
@nicolevasilevsky, I have a few comments.

File: src/ontology/mondo-edit.obo [hunk 4]
```diff
@@ -513701,6 +513701,16 @@ relationship: predisposes_towards MONDO:0013792 {source="OMIM:614519"} ! intrace
...
 
+[Term]
+id: MONDO:0100534
+name: ocular dysgenesis caused by defects in PAX6 regulation
+def: "Any eye disorder in which the cause of the disease is a variant in the PAX6 gene itself or a variant within another locus that results in defective regulation of the PAX6 gene." [https://clinicalgenome.org/affiliation/40077/, https://orcid.org/0000-0001-5935-2391, PMID:17679951, PMID:24290376, PMID:26394807]
+comment: This new term was proposed by the ClinGen Glaucoma / Neuro-Ophthalmology GCEP as a parent term for cases of ocular disease caused by variants in the PAX6 gene itself (MONDO:0800183 PAX6-related ocular dysgenesis) or by disruption of PAX6 expression by variants in the ELP4 locus (MONDO:0014937 aniridia 2) or TRIM44 locus (MONDO:0014938 aniridia 3).
+is_a: MONDO:0005328 {source="https://clinicalgenome.org/affiliation/40077/", source="https://orcid.org/0000-0001-5935-2391"} ! eye disorder
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/8620 {source="https://clinicalgenome.org/affiliation/40077/", source="https://orcid.org/0000-0001-5935-2391"} ! PAX6
```
[the remark this answers is not in the document]
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:513710]: I removed this

File: src/ontology/mondo-edit.obo [hunk 5]
```diff
@@ -518415,10 +518425,12 @@ property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/
...
 is_a: MONDO:0005328 {source="https://clinicalgenome.org/affiliation/40077/", source="https://www.clinicalgenome.org/working-groups/clinical-domain/ocular-cdwg/"} ! eye disorder
+is_a: MONDO:0100534 {source="https://clinicalgenome.org/affiliation/40077/", source="https://orcid.org/0000-0001-5935-2391"} ! ocular dysgenesis caused by defects in PAX6 regulation
```
[the remark this answers is not in the document]
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:518430]: I revised the logical axiom

**Comment by nicolevasilevsky** [comment]
ready for re-review

**Review by sabrinatoro — APPROVED** [review]
all my requests have been addressed. Approving and merging

**Pull request merged**